RELEASE_NOTES.txt

```diff
@@ -1,3 +1,19 @@
+======================================================
+-------------------------------------------------------------------------------------------
+ Version 9.0.1                                                                   
+-------------------------------------------------------------------------------------------
+======================================================
+
+
+==========================================
+ Bugfixes
+==========================================
+
+- Automatically choose correct acces token type for OIDC (#263)
+    ○ automatically choose JWT token to use as Access Token, if no other type is enforced by PiWeb Server
```

Comment:
I would change "Access Token" to "access token" since it's not a proper noun (not the OAuth Access Token).

Reply:
Done